srcpkgs/kalendar/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'kalendar'
+pkgname=kalendar
+version=0.4.0
+wrksrc="$pkgname-v$version"
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules gettext"
+makedepends="qt5-devel qt5-declarative-devel qt5-quickcontrols2-devel
+ qt5-svg-devel qt5-location-devel kirigami2-devel kdbusaddons-devel ki18n-devel
+ kcalendarcore-devel kconfigwidgets-devel kwindowsystem-devel kcoreaddons-devel
+ kcontacts-devel kitemmodels-devel kxmlgui-devel knotifications-devel
+ kiconthemes-devel kservice-devel akonadi5-devel akonadi-contacts-devel
+ calendarsupport-devel eventviews-devel"
+short_desc="Calendar application using Akonadi"
+maintainer="Michal Vasilek <michal@vasilek.cz>"
+license="GPL-3.0-or-later"
+homepage="https://invent.kde.org/pim/kalendar"
+distfiles="https://invent.kde.org/pim/kalendar/-/archive/v$version/kalendar-v$version.tar.gz"
```

Comment:
Use `${KDE_SITE}/kalendar/kalendar-${version}.tar.xz` always use the generated tarballs from `downloads.kde.org` since the git archives do not contain e.g. translations